module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,59 @@
+#include "compression.hpp"
+#include <algorithm>
+#include <iostream>
+#include <iomanip>
```

Comment:
Include should be sorted alphabetically(iomanip before iostream) and header should be in another paragraph(newline between them). There is a chapter about this in the "Powtórka #1, Q&A" lesson.